de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Sales_Customs_Invoice_Description.sql

```diff
@@ -41,7 +41,7 @@ FROM
   LEFT OUTER JOIN C_DocType dt ON i.c_doctype_id = dt.C_DocType_ID AND dt.isActive = 'Y'
   LEFT OUTER JOIN C_DocType_Trl dtt ON i.c_doctype_id = dtt.C_DocType_ID AND dtt.AD_Language = $2 AND dtt.isActive = 'Y'
   LEFT JOIN C_Customs_Invoice_Line il on il.C_Customs_Invoice_id = i.C_Customs_Invoice_ID
-  LEFT JOIN de_metas_endcustomer_fresh_reports.getShipmentAddressorNull($1) as io on 1=1
+  LEFT JOIN de_metas_endcustomer_fresh_reports.getCustomsInvoiceShipmentAddressorNull($1) as io on 1=1
```

Comment:
What's an addressor 🤓 . I hope I will never get it its way :-P (scnr, no need to update)